.github/workflows/central-publish.yml

```diff
@@ -14,8 +14,18 @@ jobs:
         with:
           distribution: 'adopt'
           java-version: 11
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew build -x check -x test
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          scan-ref: '/github/workspace/ballerina/lib'
```

Comment:
Does this module pack 3rd party dependencies? If not, this should be as follows.
```suggestion
          scan-ref: '/github/workspace/ballerina'
```
Check the other places as well.